go-controller/pkg/node/healthcheck.go

```diff
@@ -95,6 +96,22 @@ func countLocalEndpoints(ep *kapi.Endpoints, nodeName string) int {
 	return num
 }
 
+func countHostNeworkEndpoints(ep *kapi.Endpoints, nodeAddresses *sets.String) bool {
```

Comment:
this is not really counting anything, right?

Reply:
Changed it